src/main/java/models/Game.java

```diff
@@ -46,9 +46,10 @@ public Card removeTop(java.util.List<Card> l) {
         return l.remove(l.size()-1);
     }
 
-    //takes a card from the deck and puts it into a hand
-    public void deal(java.util.List<Card> hand) {
-        hand.add(removeTop(deck));
+    //takes a number of cards from the deck and puts it into a hand
```

Comment:
So we can pass in g.deal(gpHand, number_of_cards_to_draw) 

Is that was this does?